miden-crypto/src/merkle/smt/full/tests.rs

```diff
@@ -1,10 +1,12 @@
+#![cfg(feature = "std")]
 use alloc::vec::Vec;
```

Comment:
Should we instead mark the tests as `#![cfg(feature = "std")]` and import `Vec` inline?